index.html

```diff
@@ -452,6 +452,11 @@ <h3><dfn>Source</dfn> interface</h3>
     <p>
       <dfn>match()</dfn> Returns a stream that processes all quads matching the pattern.
     </p>
+    <p class="note">
+      When matching with <code>graph</code> set to <code>undefined</code> or <code>null</code>
+      it MUST match all the graphs (sometimes called <em>the union graph</em>). To match only <em>the default graph</em>
+      set <code>graph</code> to a <code>DefaultGraph</code>
```

Comment:
missing dot at the end (pedantic, but let's get this right)

Reply:
Thanks! fixed :heavy_check_mark: